style/forms/ToggleButton.scss

```diff
@@ -0,0 +1,43 @@
+.ds-form-control-toggle {
+  position: relative;
+  align-items: center;
+  display: inline-flex;
+
+  input {
+    display: none;
+  }
+
+  &__slider {
+    position: relative;
+    overflow: hidden;
+    // float: left;
+    border-radius: 37px;
+    cursor: pointer;
+    width: 26.4px;
```

Comment:
I'm a bit nervous with such numbers, do we need such precision to be accurate with wireframes?

Reply:
I do agree with Ivan here. let it be 26.

Reply:
I set it to 28px